open_dread_rando/files/custom_scenario.lua

```diff
@@ -48,15 +48,35 @@ function Scenario.InitScenario(arg1, arg2, arg3, arg4)
     init_scenario(arg1, arg2, arg3, arg4)
 
     local playerSection =  Game.GetPlayerBlackboardSectionName()
+    local currentSaveRandoHash = Blackboard.GetProp(playerSection, "THIS_RANDO_HASH")
     local randoInitialized = Blackboard.GetProp(playerSection, "RANDO_GAME_INITIALIZED")
 
+    Game.LogWarn(0, ("Cross-checking seed hashes. The current patch's hash is %q, and the current save's hash is %q."):format(Init.sThisRandoSeedHash, tostring(currentSaveRandoHash)))
```

Comment:
`"Cross-checking identifiers. The expected identifier is %q, and the current save's identifier is %q.`